src/Data/Yaml.php

```diff
@@ -24,22 +26,33 @@ class Yaml extends Handler
      */
     public static function encode($data): string
     {
-        // TODO: The locale magic should no longer be
-        //       necessary when support for PHP 7.x is dropped
+        // get YAML handler from config option
+        $handler = kirby()->option('yaml', 'spyc');
```

Comment:
What do you think about `yaml.handler` option name to standart instead just `yaml`? Like `date.handler`. Not important just idea..

Reply:
I'm thinking if we should make the YAML handler a component just like `markdown`, `smartypants` etc.

**Advantages:**

- Would be more consistent in the long run.
- No additional config option needed.

**Disadvantages:**

- During the migration phase, it wouldn't be as simple to use a specific handler.

**Possible compromise:**

We could make this a component, but also introduce a config option in 3.7 that controls which of the two default components (spyc or symfony) is activated (if not set, it would default to `spyc`). In 3.8 we would remove that option again and always use `symfony`. If users need an entirely different handler, they could register the component in a plugin. We could then provide an official plugin for Spyc for those who still need it.